src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -2371,16 +2144,16 @@ void WriteTabularLifeCycleCostReport(EnergyPlusData &state)
 
         for (iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
             rowHead(iYear) = format("{} {}", MonthNames(elcc->baseDateMonth), elcc->baseDateYear + iYear - 1);
-            tableBody(1, iYear) = RealToStr(elcc->CashFlow(costCatEnergy).yrAmount(iYear), 2);
-            tableBody(2, iYear) = RealToStr(elcc->CashFlow(costCatWater).yrAmount(iYear), 2);
-            tableBody(3, iYear) = RealToStr(elcc->CashFlow(costCatMaintenance).yrAmount(iYear), 2);
-            tableBody(4, iYear) = RealToStr(elcc->CashFlow(costCatRepair).yrAmount(iYear), 2);
-            tableBody(5, iYear) = RealToStr(elcc->CashFlow(costCatOperation).yrAmount(iYear), 2);
-            tableBody(6, iYear) = RealToStr(elcc->CashFlow(costCatReplacement).yrAmount(iYear), 2);
-            tableBody(7, iYear) = RealToStr(elcc->CashFlow(costCatMinorOverhaul).yrAmount(iYear), 2);
-            tableBody(8, iYear) = RealToStr(elcc->CashFlow(costCatMajorOverhaul).yrAmount(iYear), 2);
-            tableBody(9, iYear) = RealToStr(elcc->CashFlow(costCatOtherOperational).yrAmount(iYear), 2);
-            tableBody(10, iYear) = RealToStr(elcc->CashFlow(costCatTotOper).yrAmount(iYear), 2);
+            tableBody(1, iYear) = RealToStr(elcc->CashFlow[CostCategory::Energy].yrAmount(iYear), 2);
```

Comment:
This can be done with a loop rather than unrolled code.

Reply:
This is a bit weird since it is not looping through all the CostCategory enums and the index enums are not in the same order. We could probably divide it into three parts, right? 

            tableBody(1, iYear) = RealToStr(elcc->CashFlow[CostCategory::Energy].yrAmount(iYear), 2);
            tableBody(2, iYear) = RealToStr(elcc->CashFlow[CostCategory::Water].yrAmount(iYear), 2);
            for (int i = 0; i < 6; ++i) {
                        tableBody(i+3, iYear) = RealToStr(elcc->CashFlow[i].yrAmount(iYear), 2);
            }
            tableBody(10, iYear) = RealToStr(elcc->CashFlow[CostCategory::TotOper].yrAmount(iYear), 2);